proto/osmosis/concentrated-liquidity/position.proto

```diff
@@ -9,7 +9,7 @@ import "cosmos_proto/cosmos.proto";
 import "google/protobuf/timestamp.proto";
 import "gogoproto/gogo.proto";
 
-option go_package = "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/model";
+option go_package = "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/types";
```

Comment:
Reason: if we leave this in `model`, we cannot use it in `query.proto`, because when the code is generated, there will be an import cycle error